src/pint/data/runtime/observatories.json

```diff
@@ -0,0 +1,1355 @@
+{
+    "gbt": {
+        "tempo_code": "1",
+        "itoa_code": "GB",
+        "clock_file": "time_gbt.dat",
+        "itrf_xyz": [
+            882589.289,
+            -4924872.368,
+            3943729.418
+        ],
+        "origin": "The Robert C. Byrd Green Bank Telescope.\nThis data was obtained by Joe Swiggum from Ryan Lynch in 2021 September.\n"
+    },
+    "gbt_pre_2021": {
+        "clock_file": "time_gbt.dat",
+        "itrf_xyz": [
+            882589.65,
+            -4924872.32,
+            3943729.348
+        ],
+        "origin": "The Robert C. Byrd Green Bank Telescope.\nThe origin of this data is unknown but as of 2021 June 8 it agrees exactly with\nthe values used by TEMPO and TEMPO2.\n"
```

Comment:
This is meant to be a multiline string explaining the origin of the observatory position data. Sadly JSON appears to have no way to represent multiline strings other than one long string with `\n` in it many times. Does it make sense to write some code so that an array of strings can be supplied and then concatenated by the loading code? How human-readable/maintainable do we want this to be?

Reply:
I don't feel strongly either way.

Reply:
a related question is comments: JSON doesn't support them.  We could put in a `comment=` tag but in most cases those were all replicated in the `origin` tags so they didn't seem necessary.

Reply:
I think most of what one might want to record in a comment could go in the `origin` tag. The exception is when one might want to comment something out.